.github/ISSUE_TEMPLATE/regression.md

```diff
@@ -0,0 +1,67 @@
+---
+name: Regression
+about: Report a regression in Rust.
+labels: C-bug regression-untriaged
+---
+<!--
+Thank you for filing a regression report! 🐛 Please provide a short summary of the regression,
+along with any information you feel is relevant to replace it.
+-->
+
+### Code
+
+I tried this code:
+
+```rust
+<code>
+```
+
+I expected to see this happen: *explanation*
+
+Instead, this happened: *explanation*
+
+### Version it worked on
+
+<!--
+Provide the most recent version this worked on, for example:
+
+It most recently worked on: Rust 1.47
```

Comment:
I don't think we need both this and the text below. Maybe put 'Rust 1.47' in the `version` comment?

Reply:
Hmm, I think it would be confusing that way. Personally I would prefer to be a bit verbose rather than potentially unclear.

Reply:
Ok, seems fine.